_posts/2024-05-28-pyos-newsletter-june-2024.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: June 2024"
+excerpt: "We're starting June with a record number of Python package submissions, a call for editors, a PyCon US reflection, and some SciPy meeting news!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-june-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## <i class="fa-regular fa-comments"></i> PyCon was incredible!
+We feel so incredibly fortunate to have connected with so many of you at PyCon US! While we're still recovering from a week of summits, talks, and sprints, we wanted to share this sentiment from our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), which she shared on [her Fosstodon account](https://fosstodon.org/@leahawasser):
+<figure>
+    <a href="/images/blog/2024/may/leah-pycon.png">
+    <img src="/images/blog/2024/may/leah-pycon.png" style="max-width:100%" alt="Screenshot of a Fosstodon post from Leah Wasser, reading: In my first few moments at pycon I ran into some of my conda friends & was quickly reminded of why this conference is so powerful.  Python is more than a programming language. It's a community of people who I can geek out with about the most obscure topics. I belong. People care. They care a lot about their work. They care about helping each other learn.  While this is a tech conference, it's really people that make python the vibrant communty that it is.">
+    </a>
+</figure>
+
+During PyCon US we had over 20 people contribute to pyOpenSci at our sprint, where contributors found bugs in our [beginner-friendly Python Packaging tutorials](https://www.pyopensci.org/python-package-guide/), fixed continuing integration issues, helped with our peer review process infrastructure, and even improved our packaging guide! Keep an eye out for more in an upcoming blog post from our Executive Director and Founder, Leah Wasser!
```

Comment:
https://github.com/pyOpenSci/pyopensci.github.io/pull/398/files ready for review ✨